pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -1270,37 +1269,33 @@ scheduler_pending_pods{queue="unschedulable"} 0
 `,
 		},
 		{
-			name: "make some pods subject to backoff, add pods to unschedulableQ, and then move all to activeQ",
+			name: "add pods to unschedulableQ, and then move all to bakoffQ",
 			operations: []operation{
-				backoffPod,
 				addPodUnschedulableQ,
```

Comment:
Let's not change the output.
We could have some pods with different timestamps so that some would be in active and some in backoffq.